utils/rpms/daos.spec

```diff
@@ -2,7 +2,7 @@
 %define server_svc_name daos_server.service
 %define agent_svc_name daos_agent.service
 
-%global mercury_version 2.1.0~rc3-1%{?dist}
+%global mercury_version 2.1.0~rc4-1%{?dist}
```

Comment:
https://github.com/daos-stack/mercury/pull/68 needs to land, build and publish before this can land.